scientific_library/tvb/dsl/LEMS2python.py

```diff
@@ -66,19 +66,25 @@ def render_model(model_name, template=None):
     return model_str
 
 
-def regTVB_templating(model_filename):
+def regTVB_templating(model_filename, folder=None):
     """
     modelfile.py is placed results into tvb/simulator/models
     for new models models/__init.py__ is auto_updated if model is unfamiliar to tvb
-    file_class_name is the name of the produced file and also the class name
-
+    file_class_name is the name of the produced file and also the model's class name
+    the path to XML model files folder can be added with the 2nd argument.
+    example model files:
+        epileptort.xml
+        generic2doscillatort.xml
+        kuramotot.xml
+        montbriot.xml
+        reducedwongwangt.xml
     """
 
     # file locations
     modelfile = "{}{}{}{}".format(os.path.dirname(tvb.__file__),'/simulator/models/',model_filename.lower(),'.py')
```

Comment:
Please use 
```python
import os.path
modelfile = os.path.join(os.path.dirname(tvb.__file__), 'simulator', 'models', …..
```
since it handles the path construction correctly across platforms

Reply:
has been updated as requested